pkg/drift/drift.go

```diff
@@ -152,17 +171,40 @@ func terraformStateIAM(
 	return m, nil
 }
 
-// Finds the keys located in the left map that are missing in the right map.
-func difference(left, right map[string]*iam.AssetIAM) []string {
-	var found []string
+// differenceMap finds the keys located in the left map that are missing in the right map.
+// We return a set so that we can do future comparisons easily with the result.
+func differenceMap(left, right map[string]*iam.AssetIAM) map[string]bool {
+	found := make(map[string]bool)
```

Comment:
Prefer `struct{}` over `bool` because `struct{}` doesn't allocate (throughout this PR):

```go
found := make(map[string]struct{})
for key := range left {
  found[key] = struct{}{}
}
```

Reply:
Done